### PR TITLE
Fix for CR-1114740,CR-1114732

### PR DESCRIPTION
**Pull request by vboggara-xilinx**
#### Problem solved by the commit
Fix for CR-1114740,CR-1114732
#### Bug / issue (if any) fixed, which PR introduced the bug, how it was discovered
Generated aie reports(json) file contains invalid error message, changed with proper error message.
#### How problem was solved, alternative solutions (if any) and why they were rejected
Error messages changed to proper message.
#### Risks (if any) associated the changes in the commit
low
#### What has been tested and how, request additional testing if necessary
Tested bug cases locally with this changes.
#### Documentation impact (if any)
n/a

### CODE REVIEW

File: src/runtime_src/core/common/info_aie.cpp [hunk 1]
```diff
@@ -271,7 +271,16 @@ populate_aie_shim(const xrt_core::device *device, const std::string& desc)
...
+    pt.put("error_msg", "AIE information is not available");
+	return pt;
+  }
+  catch(const xrt_core::query::exception&) {
+	pt.put("error_msg", "AIE information is not available");
+	return pt;
+  }
+  catch (const std::exception& ex){
     pt.put("error_msg", ex.what());
     return pt;
```
Comment (stsoe) [inline src/runtime_src/core/common/info_aie.cpp:285]: I think there is a mix of tabs a spaces here, can you untabify and line up?
Please consistently add space between stmt and condition as in space between catch and parent `catch (`
 and space between condition and curly brace also.

**Review by stsoe — CHANGES_REQUESTED** [review]

File: src/runtime_src/core/common/info_aie.cpp (hunk 1 above)
Comment (vboggara-xilinx) [reply src/runtime_src/core/common/info_aie.cpp]: fixed.

File: src/runtime_src/core/common/info_aie.cpp [hunk 2]
```diff
@@ -271,6 +271,12 @@ populate_aie_shim(const xrt_core::device *device, const std::string& desc)
     std::string aie_data = xrt_core::device_query<qr::aie_shim_info>(device);
     std::stringstream ss(aie_data);
     boost::property_tree::read_json(ss, pt_shim);
+  } catch (const xrt_core::query::no_such_key&){
```
Comment (stsoe) [inline src/runtime_src/core/common/info_aie.cpp:274]:
```suggestion
  } 
  catch (const xrt_core::query::no_such_key&) {
```

File: src/runtime_src/core/common/info_aie.cpp [hunk 3]
```diff
@@ -593,6 +599,12 @@ populate_aie(const xrt_core::device *device, const std::string& desc)
     aie_data = xrt_core::device_query<qr::aie_metadata>(device);
     std::stringstream ss(aie_data);
     boost::property_tree::read_json(ss, pt_aie);
+  } catch (const xrt_core::query::no_such_key&){
```
Comment (stsoe) [inline src/runtime_src/core/common/info_aie.cpp:602]:
```suggestion
  } 
  catch (const xrt_core::query::no_such_key&) {
```

File: src/runtime_src/core/common/info_aie.cpp [hunk 4]
```diff
@@ -593,6 +599,12 @@ populate_aie(const xrt_core::device *device, const std::string& desc)
...
+    pt.put("error_msg", "AIE information is not available");
+    return pt;
+  } catch (const xrt_core::query::exception&){
```
Comment (stsoe) [inline src/runtime_src/core/common/info_aie.cpp:605]:
```suggestion
  } 
  catch (const xrt_core::query::exception&) {
```

File: src/runtime_src/core/common/info_aie.cpp [hunk 5]
```diff
@@ -593,6 +599,12 @@ populate_aie(const xrt_core::device *device, const std::string& desc)
...
+    pt.put("error_msg", "AIE information is not available");
+    return pt;
   } catch (const std::exception& ex){
```
Comment (stsoe) [inline src/runtime_src/core/common/info_aie.cpp:608]:
```suggestion
  } 
  catch (const std::exception& ex) {
```

File: src/runtime_src/core/common/info_aie.cpp (hunk 2 above)
Comment (vboggara-xilinx) [reply src/runtime_src/core/common/info_aie.cpp]: Fixed.

File: src/runtime_src/core/common/info_aie.cpp [hunk 6]
```diff
@@ -271,7 +271,16 @@ populate_aie_shim(const xrt_core::device *device, const std::string& desc)
     std::string aie_data = xrt_core::device_query<qr::aie_shim_info>(device);
     std::stringstream ss(aie_data);
     boost::property_tree::read_json(ss, pt_shim);
-  } catch (const std::exception& ex){
+  }
+  catch (const xrt_core::query::no_such_key&){
```
Comment (stsoe) [inline src/runtime_src/core/common/info_aie.cpp:275]:
```suggestion
  catch (const xrt_core::query::no_such_key&) {
```

File: src/runtime_src/core/common/info_aie.cpp [hunk 7]
```diff
@@ -271,7 +271,16 @@ populate_aie_shim(const xrt_core::device *device, const std::string& desc)
...
+    pt.put("error_msg", "AIE information is not available");
+    return pt;
+  }
+  catch (const xrt_core::query::exception&){
```
Comment (stsoe) [inline src/runtime_src/core/common/info_aie.cpp:279]:
```suggestion
  catch (const xrt_core::query::exception&) {
```

File: src/runtime_src/core/common/info_aie.cpp [hunk 8]
```diff
@@ -271,7 +271,16 @@ populate_aie_shim(const xrt_core::device *device, const std::string& desc)
...
+    pt.put("error_msg", "AIE information is not available");
+    return pt;
+  }
+  catch (const std::exception& ex){
```
Comment (stsoe) [inline src/runtime_src/core/common/info_aie.cpp:283]:
```suggestion
  catch (const std::exception& ex) {
```

File: src/runtime_src/core/common/info_aie.cpp [hunk 9]
```diff
@@ -593,7 +602,16 @@ populate_aie(const xrt_core::device *device, const std::string& desc)
     aie_data = xrt_core::device_query<qr::aie_metadata>(device);
     std::stringstream ss(aie_data);
     boost::property_tree::read_json(ss, pt_aie);
-  } catch (const std::exception& ex){
+  } 
+  catch (const xrt_core::query::no_such_key&){
```
Comment (stsoe) [inline src/runtime_src/core/common/info_aie.cpp:606]:
```suggestion
  catch (const xrt_core::query::no_such_key&) {
```

File: src/runtime_src/core/common/info_aie.cpp [hunk 10]
```diff
@@ -593,7 +602,16 @@ populate_aie(const xrt_core::device *device, const std::string& desc)
...
+    pt.put("error_msg", "AIE information is not available");
+    return pt;
+  }
+  catch (const xrt_core::query::exception&){
```
Comment (stsoe) [inline src/runtime_src/core/common/info_aie.cpp:610]:
```suggestion
  catch (const xrt_core::query::exception&) {
```

File: src/runtime_src/core/common/info_aie.cpp [hunk 11]
```diff
@@ -593,7 +602,16 @@ populate_aie(const xrt_core::device *device, const std::string& desc)
...
+    pt.put("error_msg", "AIE information is not available");
+    return pt;
+  }
+  catch (const std::exception& ex){
```
Comment (stsoe) [inline src/runtime_src/core/common/info_aie.cpp:614]:
```suggestion
  catch (const std::exception& ex) {
```

**Review by stsoe — CHANGES_REQUESTED** [review]

File: src/runtime_src/core/common/info_aie.cpp (hunk 6 above)
Comment (vboggara-xilinx) [reply src/runtime_src/core/common/info_aie.cpp]: Fixed.

**Review by chvamshi-xilinx — APPROVED** [review]

**Comment by vboggara-xilinx** [comment]
@stsoe , pl merge this pr.

**Comment by vboggara-xilinx** [comment]
Build passed, pl merge this PR?

**Comment by stsoe** [comment]
retest this please

**Comment by vboggara-xilinx** [comment]
@rozumx , pl review this PR

**Comment by stsoe** [comment]
retest this please

**Comment by vboggara-xilinx** [comment]
closing this pull request as mini pipeline failing, will submit this fix in new PR.

**Pull request closed without merging**